extensions/interactions/FractionInput/directives/FractionInput.js

```diff
@@ -122,6 +122,18 @@ oppia.directive('oppiaInteractiveFractionInput', [
               !UrlService.isIframed() &&
               WindowDimensionsService.isWindowNarrow());
           };
+
+          $scope.isAnswerValid = function() {
+            return (!$scope.FractionInputForm.$invalid && $scope.answer !== '');
+          };
+
+          $scope.$watch(function() {
+            return $scope.answer;
+          }, function(answer) {
```

Comment:
answer is unused?

Reply:
Yeah, was logging it while debugging missed it. Done.